### DISCUSSION TITLE
Requesting integration of PaperQA and Microsoft GraphRAG.

### DISCUSSION THREAD

**Discussion by vishaldwdi** [open-webui/open-webui#5651] [General]
Requesting integration of PaperQA and Microsoft GraphRAG.

# PaperQA 

-     A simple interface to get good answers with grounded responses containing in-text citations.
-     State-of-the-art implementation including document metadata-awareness in embeddings and LLM-based re-ranking and contextual summarization (RCS).
-     Support for agentic RAG, where a language agent can iteratively refine queries and answers.
-     Automatic redundant fetching of paper metadata, including citation and journal quality data from multiple providers.
-     A usable full-text search engine for a local repository of PDF/text files.
-     A robust interface for customization, with default support for all LiteLLM models.

# GraphRAG

GraphRAG is able to connect information across large volumes of information and use these connections to answer questions that are difficult or impossible to answer using keyword and vector-based search mechanisms. Building on the previous question, provide semi-technical, high-level information on how the system offers functionality for various uses. This lets a system using GraphRAG to answer questions where the answers span many documents as well as thematic questions such as “what are the top themes in this dataset?

**Comment by flefevre** [comment]
I totally agree with your proposition. The integration of PaperQA2 would definitevely help scientist in their daily task.

**Reply by flefevre** [reply]
There is already a thread here https://github.com/open-webui/open-webui/discussions/5896
